keras_nlp/models/t5/t5_backbone.py

```diff
@@ -73,6 +78,7 @@ def __init__(
         num_heads,
         hidden_dim,
         intermediate_dim,
+        key_value_dim=None,
```

Comment:
I think we need to add this to `self` and `get_config`

Reply:
should maybe also set this parameter somewhere in our unit tests so we exercise this

Reply:
Added to self and config.